community.html

```diff
@@ -0,0 +1,77 @@
+---
+title: "Community Members"
+layout: default
+comment: >-
+  Links to people involved with the CHiMaD phase field
+  community. Please
+  email <a href="mailto:daniel.wheeler@nist.gov">daniel.wheeler@nist.gov</a>
```

Comment:
Instructions for how to get added should come at the end of the page, not up top, and should outline what information to include. For example:
> Is your sketch missing from the gallery? Please e-mail [Daniel Wheeler](mailto:daniel.wheeler@nist.gov) with the following details:
> - Your name, institutional affiliation, and contact information
> - A professional photograph or preferred avatar
> - A biographical sketch concisely summarizing your engagement with phase-field methods
>
> You will be contacted for any necessary clarification, and to let you know your profile has been added.